sorc/CMakeLists.txt

```diff
@@ -140,6 +142,23 @@ message(STATUS "BUILD_GTG ............... ${BUILD_GTG}")
 message(STATUS "ENABLE_PARALLELRESTART ............... ${ENABLE_PARALLELRESTART}")
 message(STATUS "ENABLE_RRFS_WAR ...................... ${ENABLE_RRFS_WAR}")
 
+# Build RDASApp
+if(BUILD_RDASAPP)
+  list(APPEND TARGET_LIST RDASApp)
+
+  # Use build.sh directly so we stay consistent with standalone builds
+  ExternalProject_Add(RDASApp
+    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/RDASApp
+    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/RDASApp
+    # build.sh handles configure, build, and install
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND bash -c "${CMAKE_CURRENT_SOURCE_DIR}/RDASApp/build.sh -p ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR} -f -m FV3andMPAS -j 12"
```

Comment:
1. There were issues when trying to only build only FV3 via `-m FV3` hence why using `-m FV3andMPAS`.
2. `-p` specifies the install location. This puts execs in `rrfs-workflow/exec` but as `bin`, `lib`, and other directories. I think that is more of a fix needed in RDASApp and not here.